templates/node/src/services/template.ts.twig

```diff
@@ -47,15 +48,19 @@ export class {{ service.name | caseUcfirst }} {
         {%~ endif %}
         {%~ endfor %}
         const apiPath = '{{ method.path }}'{% for parameter in method.parameters.path %}.replace('{{ '{' }}{{ parameter.name | caseCamel | escapeKeyword }}{{ '}' }}', {{ parameter.name | caseCamel | escapeKeyword }}){% endfor %};
-        const payload: Payload = {};
+        const originalPayload: Params = {};
```

Comment:
Can we rename this back to just `payload`? Cant clearly see reason why it was renamed

Reply:
Because if we don't, then the compiler complains of naming conflicts